modules/virt-creating-custom-monitoring-label-for-vms.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-creating-custom-monitoring-label-for-vms_{context}"]
+= Creating a custom monitoring label for virtual machines
+
+To enable queries to multiple virtual machines from a single service, create a custom label and assign it to the virtual machine's YAML file.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* Access to the web console for stopping and restarting a virtual machine.
+
+.Procedure
+* Edit the `template` spec of your virtual machine configuration file. The following example shows the label name `metrics` being added to the label `monitor`.
++
+[source,yaml]
+----
+spec:
+  template:
+    metadata:
+      labels:
+        monitor: metrics
+----
+
+* Stop and restart the virtual machine to create a new pod with the label name given to the `monitor` label.
+
+.Verification
+
+Metrics are exposed through an HTTP service endpoint under the `/metrics` canonical name. To verify that the `node-exporter` service is working:
+
+. Obtain the `Cluster-IP` for the `node-exporter` service.
++
+[source,terminal]
+----
+$ oc get service -n <node_exporter_service_name>
+----
++
+.Example output
+[source,terminal]
+----
+NAME                  TYPE      CLUSTER-IP     EXTERNAL-IP PORT(S)  AGE
+node-exporter-service ClusterIP 172.30.226.162 <none>      9100/TCP 4d22h
```

Comment:
What if `Cluster-IP` is `None`, there should one line info or guidance is required.

My cluster showing this:
```
NAME                            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                               AGE
node-exporter                   ClusterIP   None             <none>        9100/TCP                              10d
```

Reply:
@jobbler - Please respond.

Reply:
if Cluster-IP is none, then the metrics wont work. I would check the Service definition file and make sure the type: is correct.

Reply:
My understanding - no new addition of doc content because if Cluster-IP is none, then the metrics won't be exposed.